apps/native/app/src/screens/document-detail/document-detail.tsx

```diff
@@ -393,9 +354,9 @@ export const DocumentDetailScreen: NavigationFunctionComponent<{
     }
   }, componentId)
 
-  useNavigationComponentDidAppear(() => {
+  useEffect(() => {
```

Comment:
This event did not trigger on Android development build and also not the native event Navigation.events().
 I Don't see the harm in using useEffect on mount instead, unless I am missing something 🤔